bundles/org.eclipse.jface/src/org/eclipse/jface/dialogs/DialogSettings.java

```diff
@@ -364,6 +368,30 @@ public void put(String key, String value) {
 		}
 	}
 
+	public void inputNewSearchFilter(String SearchIn) {
```

Comment:
By convention variables usually start with a lower letter.
```suggestion
	public void inputNewSearchFilter(String searchIn) {
```

Reply:
done